source/Logger.cpp

```diff
@@ -80,10 +90,51 @@ int convertSeverity(const Severity &severity)
 }
 } // namespace
 
+LogFile &LogFile::instance()
+{
+    static LogFile logFile;
+    return logFile;
+}
+
+bool LogFile::write(const std::string &line)
+{
+    std::unique_lock<std::mutex> lock{m_mutex};
+    if (!m_file.is_open())
+    {
+        return false;
+    }
+    m_file << line << std::endl;
+    return true;
+}
+
+bool LogFile::isEnabled() const
+{
+    return m_file.is_open();
+}
+
+LogFile::LogFile()
+{
+    std::string logPath{getRialtoLogPath()};
+    if (!logPath.empty())
+    {
+        // Add suffix to have rialto client and rialto ocdm logs in separate directories
```

Comment:
I think that you meant separate files not directories

Reply:
changed